src/amo/components/AuthenticateButton/index.js

```diff
@@ -9,7 +9,8 @@ import { withRouter } from 'react-router-dom';
 import { logOutFromServer, startLoginUrl } from 'amo/api';
 import { getCurrentUser, logOutUser } from 'amo/reducers/users';
 import translate from 'amo/i18n/translate';
-import Button from 'amo/components/Button';
+//import Button from 'amo/components/Button';
```

Comment:
Thanks @siddhi523 for the patch, Instead I think its better to remove the code better than commenting in this case. not sure if this is right move lets wait for more reviews ;)